acceptance_tests/features/steps/edit_respondent_details.py

```diff
@@ -48,7 +51,7 @@ def change_contact_number(_):
 
 @when('they change the email address')
 def edit_email_address(_):
-    edit_respondent_details_form.edit_email_address(Config.RESPONDENT_USERNAME)
+    edit_respondent_details_form.edit_email_address('test_respondent3@test.com')
```

Comment:
If you are going to refactor these tests you might as well add the email address as a parameter.